src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day calendar--month-length">
```

Comment:
` <div class="calendar calendar--start-day calendar--month-length">`

While indicating "calendar--start-day-wed calendar--month-length-28" your calendar should have 28 days starting from wednesday etc. Means when you change that parameters calendar should change too.